Add `preserve-parent-color` value for `forced-color-adjust` property (#37394)
mdn/content: files/en-us/web/css/@media/forced-colors/index.md

In addition to these adjustments, browsers will help ensure text legibility by drawing "backplates" behind text. This is particularly important for preserving contrast when text is placed on top of images.

There are some cases where the user agent does not force the values for the above properties:

When {{cssxref("forced-color-adjust")}} is set to `none` on an element, none of the forced color values will apply, and author styles will be applied as normal. Additionally, the backplate for text will be disabled.

> When {{cssxref("forced-color-adjust")}} is set to `preserve-parent-color` on an element, and the {{cssxref("color")}} property does not inherit from its parentl then the element will behave the same as set `preserve-parent-color` to `none`.

Author: ```suggestion
When {{cssxref("forced-color-adjust")}} is set to `preserve-parent-color` on an element, and the {{cssxref("color")}} property on the element does not inherit from its parent, then the element will behave the same as set `preserve-parent-color` to `none`.
```

> When {{cssxref("forced-color-adjust")}} is set to `preserve-parent-color` on an element, and the {{cssxref("color")}} value on the element does not inherit from its parent, then the element will behave the same as set `preserve-parent-color` to `none`.

Reviewer: ```suggestion
When {{cssxref("forced-color-adjust")}} is set to `preserve-parent-color` on an element, and the {{cssxref("color")}} value on the element does not inherit from its parent, then the element will behave the same as setting `preserve-parent-color` to `none`.
```

> When {{cssxref("forced-color-adjust")}} is set to `preserve-parent-color` on an element, and the {{cssxref("color")}} property on the element does not inherit from its parent, then the element will behave the same as set `preserve-parent-color` to `none`.

Author: ```suggestion
When {{cssxref("forced-color-adjust")}} is set to `preserve-parent-color` on an element, and the {{cssxref("color")}} value on the element does not inherit from its parent, then the element will behave the same as set `preserve-parent-color` to `none`.
```

> When a {{cssxref("system-color", "system color", "", 1)}} is specified, it will be used instead of the value that would otherwise have been forced.

Author: ```suggestion
When a [system color](/en-US/docs/Web/CSS/system-color) is specified, it will be used instead of the value that would otherwise have been forced.
```
